.github/workflows/release.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
           if [ $(echo $VERSION | grep "-") ]; then PRERELEASE=true; else PRERELEASE=false; fi
           echo "::set-output name=version::${VERSION}"
           echo "::set-output name=prerelease::${PRERELEASE}"
-      - run: echo ${{ secrets.QUAY_PASSWORD }} | docker login -u ${{ secrets.QUAY_USER }} --password-stdin quay.io
+      - run: echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u ${{ secrets.GHCR_USER }} --password-stdin
```

Comment:
@bells17 
What is `GHCR_USER`? One of GitHuB built-in secret (like `GITHUB_TOKEN`) or maintainers should set this secret?

Reply:
Sorry. I fixed it.
It seems that GITHUB_TOKEN is also set automatically when you allow deployment from Github Package to Github Actions.

- https://docs.github.com/en/actions/publishing-packages/publishing-docker-images
- https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility
- https://docs.github.com/en/actions/learn-github-actions/contexts#github-context